routers/web/repo/setting/setting.go

```diff
@@ -940,6 +940,26 @@ func SettingsPost(ctx *context.Context) {
 		log.Trace("Repository was un-archived: %s/%s", ctx.Repo.Owner.Name, repo.Name)
 		ctx.Redirect(ctx.Repo.RepoLink + "/settings")
 
+	case "private", "public":
```

Comment:
It seems that no need to define two of them. only `visibility` is enough.

Reply:
Agree